<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>867</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>YKServer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lb_host_url">
        <property name="text">
         <string>Host Url</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="le_host_url">
        <property name="text">
         <string>http://117.176.219.29:9080</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_deviceId">
        <property name="text">
         <string>deviceId(设备唯一编码)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="le_deviceId">
        <property name="text">
         <string>20220720SL4V3S001</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lb_sysCode">
        <property name="text">
         <string>sysCode(设备厂商编码)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="le_sysCode">
        <property name="text">
         <string>yikang</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lb_param">
        <property name="text">
         <string>param(提交的设备治疗最终参数)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="le_param">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lb_patientID">
        <property name="text">
         <string>patientID(患者唯一标识)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="le_patientID">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lb_recordId">
        <property name="text">
         <string>recordId(患者治疗项目的唯一标识)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QLineEdit" name="le_recordId">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="cb_auto_run">
        <property name="text">
         <string>Auto Run</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>testData</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QPushButton" name="btn_testData">
        <property name="text">
         <string>治疗方案下发测试</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>getTreatPlan</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QPushButton" name="btn_getTreatPlan">
        <property name="text">
         <string>获取计划治疗方案</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>getTreatStatus</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QPushButton" name="btn_getTreatStatus">
        <property name="text">
         <string>获取治疗状态</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>syncStatus</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="cb_dev_status"/>
      </item>
      <item row="10" column="2">
       <widget class="QPushButton" name="btn_syncStatus">
        <property name="text">
         <string>同步设备状态</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>syncTreatStatus</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QComboBox" name="cb_treat_status"/>
      </item>
      <item row="11" column="2">
       <widget class="QPushButton" name="btn_syncTreatStatus">
        <property name="text">
         <string>同步治疗状态</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>receiveDevicesData</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1" colspan="2">
       <widget class="QPushButton" name="btn_receiveDevicesData">
        <property name="text">
         <string>提交设备治疗参数</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>submitReport</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1" colspan="2">
       <widget class="QPushButton" name="btn_submitReport">
        <property name="text">
         <string>提交治疗或评定报告</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="le_url"/>
    </item>
    <item row="2" column="0">
     <widget class="QTextEdit" name="te_respond"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn_clear">
      <property name="text">
       <string>Clear Log</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="le_send"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_send">
        <property name="text">
         <string>Tcp Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QTextEdit" name="te_tcp"/>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="btn_clear_tcp">
      <property name="text">
       <string>Clear Log</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
